torch/fx/graph_module.py

```diff
@@ -66,6 +85,9 @@ def forward(self, {', '.join(free_variables)}):
         for k, v in gbls.items():
```

Comment:
I think the same bug is here too

Reply:
I guess the question is if there's any name other than `forward` we want to extract from the globals after `exec`. @zdevito  do you know?

Reply:
This can be done in a separate PR, so non-blocking